parts/k8s/windowscontainerdfunc.ps1

```diff
@@ -39,130 +39,54 @@ function Install-Containerd {
   }
 
   # TODO: check if containerd is already installed and is the same version before this.
-  $zipfile = [Io.path]::Combine($ENV:TEMP, "containerd.zip")
-  DownloadFileOverHttp -Url $ContainerdUrl -DestinationPath $zipfile
-  Expand-Archive -path $zipfile -DestinationPath $global:ContainerdInstallLocation -Force
-  del $zipfile
+  
+  # Extract the package
+  if ($ContainerdUrl.endswith(".zip")) {
+    $zipfile = [Io.path]::Combine($ENV:TEMP, "containerd.zip")
+    DownloadFileOverHttp -Url $ContainerdUrl -DestinationPath $zipfile
+    Expand-Archive -path $zipfile -DestinationPath $global:ContainerdInstallLocation -Force
+    del $zipfile
+  }elseif ($ContainerdUrl.endswith(".tar.gz")) {

```

Comment:
nitL can you format the powershell files in vs code? there are whitespace inconsistencies.

Reply:
I ran it on this file and seems like it is fixed? I ran it on some of the other files that I did not modify and there were alot of format changes not related to this PR so I think I should avoid that for this PR to keep the noise down?

Reply:
Yea, just run it on files you modified.
I think there should be a space between *}elseif (*